baseplate/experiments/providers/__init__.py

```diff
@@ -27,10 +29,14 @@ def parse_experiment(config):
         * **name**: String experiment name, should be unique for each
           experiment.
         * **owner**: The group or individual that owns this experiment.
-        * **expires**: Date when this experiment expires in ISO format
-          (YYYY-MM-DD).  The experiment will expire at 00:00 UTC on the day
-          after the specified date.  Once an experiment is expired, it is
-          considered disabled.
+        * **version**: String to identify the specific version of the
+          experiment.
+        * **start_ts**: Epoch timestamp float of date and time when you want the
```

Comment:
"A float of seconds since the epoch" would be more descriptive and then you don't need to dictionary up jargon! :)